src/components/App.tsx

```diff
@@ -747,6 +747,11 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         this.initializeScene();
       });
     }
+
+    const searchParams = new URLSearchParams(window.location.search.slice(1));
+    if (searchParams.has("web-share-target")) {
+      this.restoreFileFromShare();
+    }
```

Comment:
shouldn't this override the `initializeScene()` above instead of happen in addition?

Reply:
I'll do that 👍

Reply:
Done :)